internal/command/views/test.go

```diff
@@ -223,14 +224,18 @@ func (t *TestHuman) DestroySummary(diags tfdiags.Diagnostics, run *moduletest.Ru
 	t.Diagnostics(run, file, diags)
 
 	if state.HasManagedResourceInstanceObjects() {
-		t.view.streams.Eprint(format.WordWrap(fmt.Sprintf("\nOpenTofu left the following resources in state after executing %s, and they need to be cleaned up manually:\n", identifier), t.view.errorColumns()))
+		t.view.streams.Eprint(format.WordWrap(fmt.Sprintf("\nOpenTofu left the following resources in state after executing %s, these left-over resources can be viewed by reading the statefile written to disk(errored_test.tfstate).\n", identifier), t.view.errorColumns()))
```

Comment:
Personally - I'd still say here that "they need to be cleaned up manually", as I'm not sure it's now clear that the user should do that (Same for all other places you've changed the text)

Reply:
I am in agreement, the message does not clearly convey that the user has to manually clean up the resources after viewing the statefile. 

This is what I propose:
`OpenTofu left the following resources in state after executing %s, these left-over resources can be viewed by reading the statefile written to disk(errored_test.tfstate) and they need to be cleaned up manually:
`

WDYT @Yantrio ?